pkg/sql/parser/testdata/create_table

```diff
@@ -2260,3 +2260,15 @@ CREATE TABLE operator_tbl (a INT8 DEFAULT 1 OPERATOR(+) 2, b INT8 DEFAULT 1 OPER
 CREATE TABLE operator_tbl (a INT8 DEFAULT ((1) OPERATOR(+) (2)), b INT8 DEFAULT ((1) OPERATOR(+) (2))) -- fully parenthesized
 CREATE TABLE operator_tbl (a INT8 DEFAULT _ OPERATOR(+) _, b INT8 DEFAULT _ OPERATOR(+) _) -- literals removed
 CREATE TABLE _ (_ INT8 DEFAULT 1 OPERATOR(+) 2, _ INT8 DEFAULT 1 OPERATOR(+) 2) -- identifiers removed
+
+parse
+CREATE TABLE generated_always_as_identity_tb (
+  a INT UNIQUE,
+  b INT GENERATED ALWAYS AS IDENTITY,
+  c INT GENERATED BY DEFAULT AS IDENTITY
```

Comment:
can you also add a test to make sure it doesn't work if you try multiple GENERATED types?

for example, these shouldn't work (for both CREATE and ALTER TABLE):
```
CREATE TABLE generated_error (
   a INT UNIQUE,
   b INT GENERATED BY DEFAULT AS IDENTITY GENERATED ALWAYS AS (a) STORED
 )
-- postgres returns: both identity and generation expression specified for column "b" of table "generated_error"

CREATE TABLE generated_error (
   a INT UNIQUE,
   b INT GENERATED BY DEFAULT AS IDENTITY GENERATED ALWAYS AS IDENTITY
 )
-- postgres returns: multiple identity specifications for column "b" of table "generated_error"
```

see https://github.com/cockroachdb/cockroach/blob/6fafc569ad64c70654345dd6c685c6b32991307b/pkg/sql/parser/testdata/alter_default_privileges#L49 for an example of an error test